microcosm_fastapi/errors.py

```diff
@@ -15,4 +15,89 @@ class ErrorSchema(BaseSchema):
     message: str = "Unknown Error"
     code: int = 500
     retryable: bool
-    context: Optional[ErrorContextSchema]
\ No newline at end of file
+    context: Optional[ErrorContextSchema]
+
+
+class ParsedException:
```

Comment:
This class could also have `to_dict` method returning:
```
{
  "code": self.status_code,
  "context": self.context,
  "message": self.error_message,
  "retryable": self.retryable
}
```

Reply:
Yep this is definitely nicer. Thanks for the suggestion.